src/MapKit/MKMapCameraZoomRange.cs

```diff
@@ -0,0 +1,37 @@
+#if !WATCH // doesn't show up in watch headers
+#if XAMCORE_2_0 || !MONOMAC
+using System;
+using ObjCRuntime;
+
+namespace MapKit {
+
+	public enum  MKMapCameraZoomRange {
+		Min,
+        Max,
+	}
+
+	public partial class MKMapCameraZoomRange {
```

Comment:
I'm surprised you can have both an `enum` and a `class` of the same name ?!? (at least inside the same namespace)

Reply:
File was missing from the framework sources. Added it.